### PR TITLE
Adjust language mapping

### PR DESCRIPTION
**Pull request by rickycodes**
**Description**

Opening this as a draft for now as it relies on #3804 

This PR updates the language mappings in the project to be compatible with Crowdin. It also deletes the old translations in favour of the new ones which are more comprehensive:

image

### CODE REVIEW

File: locales/i18n.js [hunk 1]
```diff
@@ -7,35 +7,25 @@ import { LANGUAGE } from '../app/constants/storage';
...
+import id from './languages/id';
+import ja from './languages/ja';
+import ko from './languages/ko';
+import pt from './languages/pt';
+import ru from './languages/ru';
+import tl from './languages/tl';
+import vi from './languages/vi';
+import zh from './languages/zh';
+
+const supportedTranslations = { en, es, hi, id, ja, ko, pt, ru, tl, vi, zh };
```
Comment (sethkfman) [inline locales/i18n.js:22]: There are a number of new files added that are not under the `supportedTranslations`. (e.g. de, el, fr)

Comment (rickycodes) [reply locales/i18n.js:22]: ah! good catch. I'll add those 4 in.

File: locales/i18n.js [hunk 2]
```diff
@@ -7,35 +7,29 @@ import { LANGUAGE } from '../app/constants/storage';
...
-import hi from './languages/hi-in.json';
-import id from './languages/id-id.json';
-import ja from './languages/ja-jp.json';
-import ko from './languages/ko-kr.json';
-import pt from './languages/pt-br.json';
-import ru from './languages/ru-ru.json';
-import tl from './languages/tl.json';
-import vi from './languages/vi-vn.json';
-import zh from './languages/zh-cn.json';
+import en from './languages/en';
```
Comment (sethkfman) [inline locales/i18n.js:10]: Sorry to be picky but could we place these in alphabetical order?

**Review by sethkfman — APPROVED** [review]
LGTM

**Pull request merged**